### PR TITLE
Adding tests, forcefully triggering credo issues

### PR DESCRIPTION
**Pull request by psyfear**

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+engines:
+  credo:
+    enabled: true
+    channel: "beta"
```
Comment (wnuqui) [inline .codeclimate.yml:4]: Can we remove this for now? 😸

File: test/config_test.exs [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
   test "auth initialization global" do
     auth =
-      [ username: "username",
+      [
+        username: "username",
         token: "token",
         environment: "environment",
-        api_domain: "api_domain" ]
-    PromisepayEx.Config.set(auth)
+        api_domain: "api_domain",
```
Comment (wnuqui) [inline test/config_test.exs:11]: Also here, extra comma?

**Review by wnuqui — CHANGES_REQUESTED** [review]
Apart from above, LGTM 👍

File: .codeclimate.yml (hunk 1 above)
Comment (psyfear) [reply .codeclimate.yml:4]: will do!

File: test/config_test.exs (hunk 2 above)
Comment (psyfear) [reply test/config_test.exs:11]: I always put an extra comma at the end of an array (Elixir List). Coding conventions :)

File: .codeclimate.yml (hunk 1 above)
Comment (psyfear) [reply .codeclimate.yml:4]: oops removing this broke codeclimate, putting it back

**Review by wnuqui — APPROVED** [review]

**Pull request merged**